src/handlers/dkg/dkgProposals/eventHandler.ts

```diff
@@ -145,8 +145,8 @@ export async function dkgProposalEventHandler(event: SubstrateEvent) {
             blockId: eventDecoded.blockNumber,
             nonce: String(nonce),
             chainId: eventData.chainId.toString(),
-            proposalType: dkgPayloadKeyToProposalType(eventData.key),
-            data: eventData.data.toString(),
+            proposalType: dkgPayloadKeyToProposalType(eventData.kind),
+            data: '0x00',
```

Comment:
Why is the data removed now?

Reply:
@drewstone Unlike an event such as `proposal added`,  above mentioned events does not have `data` attribute in their event data.

![CleanShot 2023-06-08 at 21 44 02](https://github.com/webb-tools/webb-graphql/assets/53374218/b7f75571-4295-497a-bc10-99daa2436344)

![CleanShot 2023-06-08 at 21 39 42](https://github.com/webb-tools/webb-graphql/assets/53374218/a4a74f95-94e0-460e-8821-571f26b8b1ef)

Reply:
Sounds good, thanks for the clarification.